src/OrbitGl/FrameTrack.cpp

```diff
@@ -62,7 +62,7 @@ float FrameTrack::GetAverageBoxHeight() const {
   if (scale_factor == 0.f) {
     return 0.f;
   }
-  return box_height_ / box_height_normalizer;
+  return GetBoxHeight() / box_height_normalizer;
```

Comment:
See my previous comments on `FrameTrack::GetMaximumBoxHeight`

Reply:
This function will be refactored completely